README.md

```diff
@@ -68,7 +69,11 @@ To use the deployed Jar file with a [docker-cantaloupe](https://github.com/uclal
 For instance, on a Linux machine:
 
     docker run -p 8182:8182 -e "CANTALOUPE_ENDPOINT_ADMIN_SECRET=secret" -e "CANTALOUPE_ENDPOINT_ADMIN_ENABLED=true" \
-      -e "AUTH_COOKIE_SERVICE=https://example.com/cookie" -e "AUTH_TOKEN_SERVICE=https://example.com/token"
+      -e "AUTH_ACCESS_MODE_SERVICE=https://example.com/access" \
+      -e "AUTH_COOKIE_SERVICE=https://example.com/cookie" \
+      -e "AUTH_TOKEN_SERVICE=https://example.com/token" \
+      -e "SINAI_AUTH_TOKEN_SERVICE=http://example.com/token/sinai" \
```

Comment:
I think you want HTTPS in this URL

Reply:
Ah yes, would be good if they were consistent.